SpotlightDownloader/Spotlight.cs

```diff
@@ -54,30 +56,31 @@ public enum ApiVersion
         /// <param name="apiver">Version of the Spotlight API</param>
         /// <returns>Raw JSON response</returns>
         /// <exception cref="System.Net.WebException">An exception is thrown if the request fails</exception>
-        private static string PerformApiRequest(bool maxres, string locale = null, ApiVersion apiver = ApiVersion.v4)
+        private static readonly HttpClient httpClient = new();
+
+        private static async Task<string> PerformApiRequestAsync(bool maxres, string locale = null, ApiVersion apiver = ApiVersion.v4)
         {
-            WebClient webClient = new WebClient();
             CultureInfo currentCulture = CultureInfo.CurrentCulture;
-            RegionInfo currentRegion = new RegionInfo(currentCulture.Name);
-            string region = currentRegion.TwoLetterISORegionName.ToLower();
+            RegionInfo currentRegion = new(currentCulture.Name);
+            string region = currentRegion.TwoLetterISORegionName.ToLower(currentCulture);
```

Comment:
```suggestion
            string region = currentRegion.TwoLetterISORegionName.ToLowerInvariant();
```

Reply:
It seems like using ToUpperInvariant() is more correct here per CA1308 rule.

Reply:
Hmm. Since the API normalizes to lowercase automatically, I think we might not need to normalize case by ourselves at all. Will think about it later.